roles/mesos/templates/zk.j2

```diff
@@ -1,14 +1,3 @@
-{% if mesos_zk_url is defined %}
-{{ mesos_zk_url }}
-{% else %}
-{%- set mesos_zk_url = '' -%}
-{%- for host in groups[mesos_zookeeper_group] -%}
-  {%- if not loop.last -%}
-    {%- set mesos_zk_url = mesos_zk_url + hostvars[host]['ansible_ssh_host'] + ':2181,' -%}
-  {%- endif -%}
-{%- if loop.last -%}
-  {%- set mesos_zk_url = mesos_zk_url + hostvars[host]['ansible_ssh_host'] + ':2181/mesos' -%}
-zk://{{ mesos_zk_url }}
-  {%- endif -%}
-{%- endfor -%}
+{% if zookeeper_peers_nodes is defined %}
```

Comment:
Is there anycase where zookeeper_peers_nodes is not defined? I think it should be always defined, otherwise this should raise an error.